fdbclient/VersionVector.h

```diff
@@ -30,14 +30,20 @@
 #include "fdbclient/FDBTypes.h"
 
 struct VersionVector {
-	std::unordered_map<Tag, Version> versions;
+	std::map<Tag, Version> versions;
 	Version maxVersion; // Specifies the max version in this version vector. (Note:
 	                    // there may or may not be a corresponding entry for this
 	                    // version in the "versions" map.)
 
 	VersionVector() : maxVersion(invalidVersion) {}
 	VersionVector(Version version) : maxVersion(version) {}
 
+private:
+	inline void setVersionNoCheck(const Tag& tag, Version version) {
+		versions[tag] = version;
```

Comment:
To make it consistent with "setVersion()" would you please do "ASSERT(tag != invalidTag);" before this line?

Reply:
And, do "if (version > maxVersion) {
			maxVersion = version;
		}" after this line?

Reply:
I intentionally avoid the assertion and updating maxVersion here. The reasoning is that this is called from `getDelta`, which iterates through an existing VV that have been validated before.

Reply:
Ok, makes sense. Would you please add a comment to the effect that this is not to be used outside the get()/apply() methods? Thanks!